### DISCUSSION TITLE
Easylist format exports, to use within a proxy

### DISCUSSION THREAD

**Discussion by v1gnesh** [uBlockOrigin/uAssets#23534] [Ideas]
It's a long-time wishlist item to run something as powerful as uBO for the whole network.
In a step towards that direction, it seems that we can use easylist format block lists within the [newly released 1.0 version of PolarProxy](https://www.netresec.com/?page=Blog&month=2024-05&post=PolarProxy-1-0-Released).

Here's an example of how a listing is used - https://github.com/Netresec/PolarProxy/blob/59e14543cbc0b9fc4990016a1eac0eb5bfe86110/rulesets/ruleset-block-malicious.json#L121-L134

If all uBO customization is in the form of static filter rules (custom & rule lists),
a) Are they in easylist format, in general?
b) if 'a' is true, I should be able to export the custom static filter rules as a list, and use in PolarProxy, right?

**Comment by ghost** [comment]
Bad idea. Pi-hole figured it out 5y ago https://github.com/pi-hole/pi-hole/pull/2912

**Reply by garry-ut99** [reply]
> gwarser : Bad idea

Yes.

> v1gnesh : Easylist format exports, to use within a proxy

Possible but risky: pulling out too many hostnames from ad-blocker extensions and then inserting them directly into DNS/hosts will block more ads, but also will cause more false hits and breakages, the answer why it happens is here: https://discourse.pi-hole.net/t/convert-easylist-to-acceptable-format/42247

But if you still want to take a risk, you can try some pre-parsed lists:
[Easylist format exports, to use within a proxy](https://github.com/ZingyAwesome/easylists-for-pihole)

**Reply by v1gnesh** [reply]
Okay, thank you both.
Wasn't there a uBO-core or similar some years ago, that allowed you to run static filter lists within nodejs (or something similar).
It would be really nice to have the power of uBO network-wide.

**Reply by garry-ut99** [reply]
Yes, it's here:
- readme: https://github.com/gorhill/uBlock/tree/master/platform/nodejs#ublock-origin-core
- related discussion: https://github.com/uBlockOrigin/uBlock-issues/issues/1664
